src/main/java/dev/galacticraft/mod/content/block/machine/SimpleMultiBlockMachineBlock.java

```diff
@@ -51,6 +56,21 @@ protected SimpleMultiBlockMachineBlock(Properties properties, ResourceLocation f
         this.partState = partBlock.defaultBlockState();
     }
 
+    @Override
+    protected void onExplosionHit(BlockState state, Level world, BlockPos pos, Explosion explosion, BiConsumer<ItemStack, BlockPos> stackMerger) {
```

Comment:
It would be better to implement `wasExploded` instead of `onExplosionHit`, as there are explosions that do not break blocks.